### ISSUE TITLE
How to insert symbol using keybd.press

### ISSUE THREAD

**Issue by kevinho1996** [ChiHangChen/KeyboardSimulator#8]
symbol includes _ " ; , .

**Comment by jeffkeller87** [comment]
These keys aren't supported yet, but I hope to add support in the future. Take a look at `?keyboard_value` for a list of supported keys.

**Comment by JohannesEmm** [comment]
HI, good point. Also the @ symbos seems not working, very important for email addresses. Is there a workaround? not yet sure what the keyboard_value table shoudjl be used for.

**Comment by ChiHangChen** [comment]
> HI, good point. Also the @ symbos seems not working, very important for email addresses. Is there a workaround? not yet sure what the keyboard_value table shoudjl be used for.

For some symbols you can try to use `keybd.type_string` instead of `keybd.press`
Let me know if this function is still not working.

**Comment by JohannesEmm** [comment]
Yes tried both. The type string function prints a " instead of the @.
Maybe just for the next version.
Great package btw!

**Comment by ChiHangChen** [comment]
I tried on my Win10 machine seems work

![image](https://github.com/ChiHangChen/KeyboardSimulator/assets/22412708/ae772095-c72c-4db9-b693-e430f90f2521)

Can you provide your system information and R information?

**Comment by JohannesEmm** [comment]
Hi OK, maybe a language issue as have an Italian keyboard  R 4.2.1.
